src/pages/SocialGraph.vue

```diff
@@ -122,11 +139,48 @@ export default {
     return {
       type: 'collector',
       account: EXAMPLE_CREATOR_ADDRESS,
-      responseType: '',
-      response: [],
+      currentPage: 1,
```

Comment:
Is the page index start with 0 better?

Reply:
I think `1` is better in the context that "it's the first page of data to view"?

Reply:
is this value displayed? prefer `0` in code/data, and +1 when displaying

Reply:
Yes, this one is displayed here:
https://github.com/likecoin/likecoin-nft-dashboard/blob/aa58212a97cad56fd68ca4dc318dd90366e8670c/src/pages/SocialGraph.vue#L31
so is it more like a UI prop than a data prop?

Reply:
i think it's better for data to use `currentPage: 0` and add a computed var `displayPage = currentPage + 1`
but let's not drag on this for too long...